lib/parse.js

```diff
@@ -530,10 +534,14 @@ function createParser({
           return;
         }
 
+        // TODO: Suggest the item with the smallest distance or even multiple results?
+        // Implementation note: swtich to Levenshtein automaton variation.
+        const maybeSuggestion = keysOfParser.find(metaKey => LevenshteinDistanceWithMax(metaKey, state.key, MAX_EDIT));
+
         // throw
         throw new ParseError({
           code: 'unknownMeta',
-          args: [state.key],
+          args: [state.key, maybeSuggestion],
           message: `Unknown metadata: @${state.key}`,
```

Comment:
Don't revert the change to the message. We put the variable into `vars` if it is used to compose `message`.